### PR TITLE
docs - add guc optimizer_use_gpdb_allocators

### PR DESCRIPTION
**Pull request by mkiyama**

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -7503,6 +7506,38 @@
       </table>
     </body>
   </topic>
+  <topic id="optimizer_use_gpdb_allocators">
+    <title>optimizer_use_gpdb_allocators</title>
+    <body>
+      <p>When GPORCA is enabled (the default) and this parameter is <codeph>true</codeph> (the
+        default), GPORCA uses Greenplum Database memory management when executing queries. When set
+        to <codeph>false</codeph>, GPORCA uses GPORCA-specific memory management.</p>
```
Comment (lenapivotal) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: My only thought is that I believe this guc might not True by default in pre 6.0 versions (but I could be wrong, I think it's not on by default in 5.22...)

Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: I think a customer would then wonder what the benefit of using one vs the other is.  Does using GPDB memory management mean GPORCA would be constrained by resource groups/queues, whereas with GPORCA-specific memory it gets as much as it needs and could possibly compete with gpdb memory?

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 2]
```diff
@@ -7503,6 +7506,38 @@
       </table>
     </body>
   </topic>
+  <topic id="optimizer_use_gpdb_allocators">
+    <title>optimizer_use_gpdb_allocators</title>
+    <body>
+      <p>When GPORCA is enabled (the default) and this parameter is <codeph>true</codeph> (the
+        default), GPORCA uses Greenplum Database memory management when executing queries. When set
```
Comment (vraghavan78) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: yes... the code say it is false. Need to fix that in the code. Let me please get back to u.

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml (hunk 1 above)
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: added benifits of GPDB query mem. mgmt.

Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: See PR for guc default change to true. https://github.com/greenplum-db/gpdb/pull/8771

**Pull request merged**